lib/mandriller.rb

```diff
@@ -1,6 +1,6 @@
+require 'action_mailer'
+require 'multi_json'
 require 'mandriller/version'
 require 'mandriller/errors'
-require 'mandriller/base'
-
-module Mandriller
-end
+require 'mandriller/settings_methods'
+require 'mandriller/base'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.